packages/truffle-5/truffle-config.js

```diff
@@ -0,0 +1,119 @@
+const homedir = require('os').homedir
+const path = require('path')
+
+const HDWalletProvider = require('@truffle/hdwallet-provider')
+
+const DEFAULT_MNEMONIC =
+  'explain tackle mirror kit van hammer degree position ginger unfair soup bonus'
+
+const defaultRPC = network => `https://${network}.eth.aragon.network`
+
+const configFilePath = filename => path.join(homedir(), `.aragon/${filename}`)
+
+const mnemonic = () => {
+  try {
+    return require(configFilePath('mnemonic.json')).mnemonic
+  } catch (e) {
+    return DEFAULT_MNEMONIC
+  }
+}
+
+const settingsForNetwork = network => {
+  try {
+    return require(configFilePath(`${network}_key.json`))
+  } catch (e) {
+    return {}
+  }
+}
+
+// Lazily loaded provider
+const providerForNetwork = network => () => {
+  let { rpc, keys } = settingsForNetwork(network)
+  rpc = rpc || defaultRPC(network)
+
+  if (!keys || keys.length === 0) {
+    return new HDWalletProvider(mnemonic(), rpc)
+  }
+
+  return new HDWalletProvider(keys, rpc)
+}
+
+const mochaGasSettings = {
+  reporter: 'eth-gas-reporter',
+  reporterOptions: {
+    currency: 'USD',
+    gasPrice: 3,
+  },
+}
+
+const mocha = process.env.GAS_REPORTER ? mochaGasSettings : {}
+
+module.exports = {
+  networks: {
+    rpc: {
+      network_id: 15,
+      host: 'localhost',
+      port: 8545,
+      gas: 6.9e6,
+      gasPrice: 15000000001,
+    },
+    devnet: {
```

Comment:
Let's remove `devnet` from this config and add it separately when we need to (e.g. in dao-templates)